src/GafferScene/RenderController.cpp

```diff
@@ -1107,6 +1109,7 @@ class RenderController::SceneGraph
 
 		IECoreScenePreview::Renderer::ObjectInterfacePtr m_boundInterface;
 		bool m_expanded;
+		VisibleSet::Result::DrawMode m_drawMode;
```

Comment:
Does it make sense to replace `m_expanded` and `m_drawMode` with a single `VisibleSet::Result m_visibility` field?

Reply:
As discussed, `m_visibility` felt like it could be easily confused with scene visibility and we waffled a bit on a better name, so for now I've renamed `m_expanded` to `m_descendantsVisible` in 9c070fb to match the naming within `VisibleSet::Visibility` so we're a little more clear on the direct usage.